client.go

```diff
@@ -252,6 +253,35 @@ func (c *Client) CreateDump() (resp *TaskInfo, err error) {
 	return resp, nil
 }
 
+func (c *Client) MultiSearch(queries []SearchRequest) (*MultiSearchResponse, error) {
```

Comment:
The user should provide `{ queries:  []SearchRequest }` not just ` []SearchRequest`

Reply:
Yes, but it is much less annoying for the user to send a search request table than to create a map to put his table in. So I do it for them

Reply:
It's up to you! The idea of adding it in the object is so that if in the future their are other fields than `queries` you don't have to do a breaking change